aeon/clustering/deep_learning/_ae_fcn.py

```
@@ -201,6 +205,11 @@ def build_model(self, input_shape, **kwargs):
         import numpy as np
         import tensorflow as tf
 
+        if self.metrics is None:
+            metrics = ["mean_squared_error"]
+        else:
+            metrics = self.metrics
```

the code should check if the input is not a list and put it in a list, as keras only takes input list for metrics. if the user gives a list it should stay a list, if its something else the code should raise a value error